python/cugraph/setup.py

```diff
@@ -12,28 +12,15 @@
 # limitations under the License.
 
 import os
-import sys
-import sysconfig
 import shutil
 
-# Must import in this order:
-#   setuptools -> Cython.Distutils.build_ext -> setuptools.command.build_ext
-# Otherwise, setuptools.command.build_ext ends up inheriting from
-# Cython.Distutils.old_build_ext which we do not want
-import setuptools
-
-try:
-    from Cython.Distutils.build_ext import new_build_ext as _build_ext
-except ImportError:
-    from setuptools.command.build_ext import build_ext as _build_ext
-
-from distutils.sysconfig import get_python_lib
-
-import setuptools.command.build_ext
-from setuptools import find_packages, setup, Command
-from setuptools.extension import Extension
+from setuptools import find_packages, Command
+from skbuild import setup
 
 from setuputils import get_environment_option
+# from setuputils import get_cuda_version_from_header
```

Comment:
There's a lot of FIXMEs in this file. Could you resolve them as best as you can, then I can help if there are any outstanding issues?

Reply:
Addressed